tests/aequilibrae/project/test_network.py

```diff
@@ -7,31 +7,25 @@
 from aequilibrae.project import Project
 from aequilibrae.project.network.network import Network
 from aequilibrae.parameters import Parameters
+from os.path import join, dirname
 from warnings import warn
 from random import random
+from aequilibrae.project.spatialite_connection import spatialite_connection
 
 
 class TestNetwork(TestCase):
     def setUp(self) -> None:
+        spatialite_folder = dirname(dirname(dirname(dirname(os.path.abspath(__file__)))))
```

Comment:
I wonder if there's a way to do this nicer, it's used in all of the tests and I don't like it. Having it in one place would also make it easier to change.

Reply:
in rfmd i put everything into a spatialite_utils.py file that had an ```ensure_spatialite_on_path``` function that would add it if it didn't exist

Reply:
```python
def add_mod_spatialite_path():
    """Ensure that the spatialite dll is on the path."""
    root_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "..", ".."))
    spatialite_dir = os.path.join(root_dir, "bin", "mod_spatialite-4.3.0a-win-amd64")
    if spatialite_dir not in os.environ["PATH"]:
        os.environ["PATH"] = spatialite_dir + ";" + os.environ["PATH"]
```

Reply:
The problem is that the path to Spatialite is not always the same. When running the tests on GitHub, Spatialite needs to be downloaded. I changed the way it happens so it is a little more elegant